tests/cnf/core/network/sriov/tests/externalllymanaged.go

```diff
@@ -244,7 +246,9 @@ var _ = Describe("ExternallyManaged", Ordered, Label(tsparams.LabelExternallyMan
 					"MaxTxRate and VlanId configurations have not been reverted to the initial one")
 
 				By("Configure SR-IOV with flag ExternallyManaged true")
-				createSriovConfiguration(sriovAndResourceNameExManagedTrue, sriovInterfacesUnderTest[0], true)
+				err = sriovenv.CreateSriovNmStateConfiguration(sriovAndResourceNameExManagedTrue, sriovInterfacesUnderTest[0], true)
+				Expect(err).ToNot(HaveOccurred(),
+					fmt.Sprintf("Failed to create sriov configuration via nmstate %s", err))
 			})
```

Comment:
wrong discription and remove err:
Expect(err).ToNot(HaveOccurred(),"Failed to create sriov configuration with flag ExternallyManaged true")

Reply:
changed